alphastats/gui/utils/overview_helper.py

```diff
@@ -21,47 +20,17 @@ def get_intensity_distribution_processed():
     return st.session_state[StateKeys.DATASET].plot_sampledistribution()
 
 
-# @st.cache_data  # TODO check if caching is sensible here and if so, reimplement with dataset-hash
-def get_display_matrix():
-    processed_df = pd.DataFrame(
-        st.session_state[StateKeys.DATASET].mat.values,
-        index=st.session_state[StateKeys.DATASET].mat.index.to_list(),
-    ).head(10)
-
-    return processed_df
-
-
 def display_matrix():
-    text = (
-        "Normalization: "
-        + str(
-            st.session_state[StateKeys.DATASET].preprocessing_info[
-                PreprocessingStateKeys.NORMALIZATION
-            ]
-        )
-        + ", Imputation: "
-        + str(
-            st.session_state[StateKeys.DATASET].preprocessing_info[
-                PreprocessingStateKeys.IMPUTATION
-            ]
-        )
-        + ", Log2-transformed: "
-        + str(
-            st.session_state[StateKeys.DATASET].preprocessing_info[
-                PreprocessingStateKeys.LOG2_TRANSFORMED
-            ]
-        )
-    )
```

Comment:
I think it would be valid to have a textual representation of the preprocessing here, but I agree with deleting the version as it was. How about we assemble a 'method section' based on the preprocessing parameters?

Reply:
I found this also highly redundant to the display of preprocessing parameters on the sidebaer. .. a human-readable "Methods" section (that can be pasted into a paper) would be awesome!